src/content/config.ts

```diff
@@ -6,6 +6,13 @@ const personSchema = {
     company: z.string().optional(),
     title: z.string(),
     imgName: z.string(),
+    links: z
+      .object({
+        twitter: z.string().optional(),
+        github: z.string().optional(),
+        website: z.string().optional(),
```

Comment:
just figured we should add linkedin, especially for the organizers profiles, most of them don't have github/twitter